jquery.marquee.js

```diff
@@ -121,8 +121,10 @@
                 }
             });
 
-            // since speed option is changed to duration, to support speed for those who are already using it
-            o.duration = o.speed || o.duration;
+            // Reintroduce speed as an option. It calculates duration as a factor of the container width
+            if (o.speed) {
+                o.duration = o.speed * parseInt($this.width(), 10);
```

Comment:
Lol I knew it wasn't quite right but at the time it did the trick for me :) Thanks for the science. This means it's closer to actually being accepted. I'll update the code here.